files/bash_aliases

```diff
@@ -25,6 +25,10 @@ alias dwp='cd ~/work/dwp/'
 alias yfw='cd ~/work/yapp-framework/'
 alias yiicore='cd ~/work/yapp-framework/vendor/yiisoft/yii2/'
 
+## Git
+alias gs="git status"
```

Comment:
I seems that both `gs` and `gc` are some kind of executable on mac (and therefore probably linux). What do you think, could we pick an alias that does not collide with some existing command.

Reply:
Sure!